test-tools/proof-explorer/src/wallet-connection.tsx

```diff
@@ -0,0 +1,214 @@
+import { detectConcordiumProvider, WalletApi } from '@concordium/browser-wallet-api-helpers';
+import { CredentialStatements, HexString, VerifiablePresentation } from '@concordium/web-sdk';
+import { SessionTypes, SignClientTypes } from '@walletconnect/types';
+import SignClient from '@walletconnect/sign-client';
+import QRCodeModal from '@walletconnect/qrcode-modal';
+import EventEmitter from 'events';
+
```

Comment:
Added this now.

Reply:
![image](https://github.com/Concordium/concordium-web3id/assets/105212/89dc575b-346c-473c-8f32-2fe86c209ac3)

Reply:
Just nitpicking: Since the two buttons look weird side by side. Myabe move the `not connected to wallet/connected to the wallet` to a section just below the `connect browser/connect mobile` section.

Reply:
That makes sense.

Reply:
I have moved this around a bit.